BraveWallet/Crypto/BuySwapSwap/SendTokenView.swift

```diff
@@ -22,11 +23,18 @@ struct SendTokenView: View {
   @ScaledMetric private var length: CGFloat = 16.0
   
   private var isSendDisabled: Bool {
-    guard let sendAmount = Double(amountInput), let balance = sendTokenStore.selectedSendTokenBalance else {
+    guard let sendAmount = BDouble(amountInput),
```

Comment:
Can you validate that this doesn't crash if you enter alpha-numeric characters? I remember having a crash when I had `BDouble(value, radix: 16)` where value included non-hex digits